gateway/gateway.go

```diff
@@ -405,19 +419,26 @@ func (g *gateway) receivePackets(ctx context.Context) {
 				if isDuplicate {
 					continue
 				}
+
+				minSNR := sfToSNRMin[int(packets[i].datarate)]
+				if float64(packets[i].snr) < minSNR {
+					g.logger.Debugf("packet skipped due to low signal noise ratio")
```

Comment:
talked offline about maybe doing this check in other spots, or moving it to a `Warn`. olivia gonna try some things